esrally/track/params.py

```diff
@@ -698,9 +789,14 @@ def create_readers(num_clients, start_client_index, end_client_index, corpora, b
             offset, num_docs, num_lines = bounds(docs.number_of_documents, start_client_index, end_client_index,
                                                  num_clients, docs.includes_action_and_meta_data)
             if num_docs > 0:
-                logger.info("Task-relative clients at index [%d-%d] will bulk index [%d] docs starting from line offset [%d] for [%s/%s] "
+                target = "/"
+                if docs.target_index:
+                    target = f"{docs.target_index}/{docs.target_type}"
```

Comment:
Oneline the three lines like:

`target = f"{docs.target_index}/{docs.target_type}" if docs.target_index else "/"`?